sx126x/sx126x.go

```diff
@@ -34,8 +34,9 @@ const (
 )
 
 const (
-	PERIOD_PER_SEC  = (uint32)(1000000 / 15.625) // SX1261 DS 13.1.4
-	SPI_BUFFER_SIZE = 256
+	PERIOD_PER_SEC      = (uint32)(1000000 / 15.625) // SX1261 DS 13.1.4
+	SPI_BUFFER_SIZE     = 256
+	RADIOEVENTCHAN_SIZE = 1
```

Comment:
The spi buffers are pre-allocated to avoid heap allocation when in interrupt. 
Most of the time, this rx/tx buffer slices are small (<10bytes)

In sx126x, the RX spi buffer is also used to bulk read packets payloads (which can be >10bytes).
I need to spend more time on this part of the code to optimize the spi buffer size. 

I'll switch the PR to draft until it's done

Reply:
Anything I can do to help get this finished @ofauchon ?

Reply:
Hi @deadprogram, 

I was not very happy with using static arrays for SPI rx/tx (instead of slices). 
I could not find a solution to avoid this 256 byte SPI_BUFFER_SIZE in sx126x.
 
I just rebased the code to upstream/dev, and it works properly on LoraE5. 
So I guess you should merge it as it fixes the problem.

Reply:
Thank you for working on it @ofauchon do you want to make this PR "ready for review" now?